pkg/sippyserver/pr_commenting_processor.go

```diff
@@ -555,6 +543,59 @@ func (aw *AnalysisWorker) processRiskAnalysisComment(prPendingComment models.Pul
 	log.Debugf("Comment added to pendingComments: %s/%s/%s", pendingComment.org, pendingComment.repo, pendingComment.sha)
 }
 
+func buildComment(sortedAnalysis RiskAnalysisEntryList, sha string) string {
+	var sb strings.Builder
+	sb.WriteString(fmt.Sprintf("Job Failure Risk Analysis for sha: %s\n\n| Job Name | Failure Risk |\n|:---|:---|\n", sha))
+
+	for _, value := range sortedAnalysis {
+		tableKey := value.Key
+
+		if value.Value.URL != "" {
+			tableKey = fmt.Sprintf("[%s](%s)", value.Key, value.Value.URL)
+		}
+
+		var riskSb strings.Builder
+		riskSb.WriteString(fmt.Sprintf("<b>%s</b>", value.Value.RiskLevel.Name))
+		for i, t := range value.Value.TestRiskAnalysis {
+			if i > 3 {
+				break
+			}
```

Comment:
GitHub [appears to have a limit of 65536 chars for a comment](https://github.com/orgs/community/discussions/41331).  I see you limit it to 3 tests per job, but if a PR has a ton of jobs, is there still a risk we might exceed it?

Reply:
Hm, it may not be worth considering.  I'd guess it's max 3-4,000 chars for a job with 3 failed tests, so you'd have to have like 16+ of them

Reply:
Great point.  I can think of a few ways to protect against this.  Make the limits a variable that gets passed in and if the returned comment is over X lower the limits.  If our limit is 1 and we are still over then only show High Risks.  Something like that.